docs/_content/ontwikkelaars/aan-de-slag.md

```diff
@@ -290,13 +290,13 @@ $ docker-compose down
 
 De API's en API documentatie zijn beschikbaar op de volgende URLs:
 
-* `http://localhost:800x/api/v1/` - API root
-* `http://localhost:800x/api/v1/schema/` - API documentatie
+* `http://<ip-of-localhost>:800x/api/v1/` - API root
+* `http://<ip-of-localhost>:800x/api/v1/schema/` - API documentatie
 
 
 ### API Guides
 
-Er zijn verschillende [API guides](guides) beschikbaar met
+Er zijn verschillende [API guides](https://ref.tst.vng.cloud/ontwikkelaars/guides) beschikbaar met
```

Comment:
We'd prefer a relative URL here too :+1:

Reply:
Thanks! 💯 Reverted.